lib/openshift/machine_set.rb

```diff
@@ -40,5 +40,10 @@ def taints(user: nil, cached: true, quiet: false)
       raw_resource(user: user, cached: cached, quiet: quiet).
         dig('spec', 'taints')
     end
+
+    def windows_machinesets(user: nil, cached: true, quiet: false)
```

Comment:
Anything indicate it's a windows machineset ?

Reply:
The naming and definition of the method can be more straightforward, I suggest defining a method like `is_windows_machineset?` which returns true or false.

```
def is_windows_machineset?(user: nil, cached: true, quiet: false)
  rr = raw_resource(user: user, cached: cached, quiet: quiet)
  rr.dig('spec', 'template', 'metadata', 'labels','machine.openshift.io/os-id') == "Windows"
end
```

Reply:
@jhou1 , I fully agree with you. Also related to what we spoke about earlier.